components/supervisor/frontend/src/index.ts

```diff
@@ -295,3 +299,35 @@ LoadingFrame.load().then(async (loading) => {
         //#endregion
     })();
 });
+
+const gitpodHost = serverUrl.url.host;
+async function maybeRedirectToCustomUrl(info: IDEFrontendDashboardService.Info) {
+    const isDataOps = await experimentsClient.getValueAsync("dataops", false, {
+        user: { id: info.loggedUserId },
+        gitpodHost,
```

Comment:
push a nit commit to add `gitpodHost` field, since I saw @jeanp413 was tring to enable for current host